tests/system/test_requests.py

```diff
@@ -129,3 +129,46 @@ def test_ok(self):
         transactions = self.get_transaction_payload()
         r = requests.post(self.transactions_url, json=transactions)
         assert r.status_code == 202, r.status_code
+
+
+class CorsTest(CorsBaseTest):
+
+    def test_ok(self):
+        transactions = self.get_transaction_payload()
+        r = requests.post(self.transactions_url, json=transactions, headers={'Origin': 'http://www.elastic.co'})
+        assert r.status_code == 202, r.status_code
```

Comment:
Could use an assert of the `Access-Control-Allow-Origin == 'http://www.elastic.co'`